week3/src/App.js

```diff
@@ -0,0 +1,87 @@
+import logo from "./logo.svg";
+import { useState } from "react";
+import "./App.css";
+import FightGame from "./Fight";
+import CookJjin from "./img/CookJjin.png";
+import CoolJjin from "./img/CoolJjin.png";
+import CryJjin from "./img/CryJjin.png";
+import FuckJjin from "./img/FuckJjin.png";
+import HyumJjin from "./img/HyumJjin.png";
+import MacJjin from "./img/MacJjin.png";
+import ShyJjin from "./img/ShyJjin.png";
+import TropyJjin from "./img/TropyJjin.png";
+
+const fighterImgs = [
+  {
+    img: CookJjin,
+    name: "👨🏻‍🍳요리하는 남자👨🏻‍🍳 찌니💛",
+  },
+  {
+    img: CoolJjin,
+    name: "🐂황소같은 남자🐂 찌니💛",
+  },
+  {
+    img: CryJjin,
+    name: "💧눈물 흘리는 남자💧 찌니💛",
+  },
+  {
+    img: HyumJjin,
+    name: "먼혐지니👁👁",
+  },
+  {
+    img: MacJjin,
+    name: "💻맥북 가진남자💻 찌니💛",
+  },
+  {
+    img: ShyJjin,
+    name: "👀부끄러운 남자👀 찌니💛",
+  },
+  {
+    img: TropyJjin,
+    name: "🏆상받는 남자🏆 찌니💛",
+  },
+  {
+    img: FuckJjin,
+    name: "🎸뻐킹 멋진남자🎸 찌니💛",
+  },
+];
+
+window.onload = function () {
+  fighterImgs.sort(() => Math.random() - 0.5);
+};
+function App() {
+  window.onload();
+
+  const [fighter, setFighter] = useState(fighterImgs);
+
+  const [winImg, setWinImg] = useState([]);
+
+  const [round, setRound] = useState(0);
+
+  const eventHandle = (e) => {
+    setWinImg([
+      ...winImg,
+      {
+        img: e.target.src,
+        name: e.target.name,
+      },
+    ]);
+    setFighter(fighter.slice(2));
```

Comment:
```suggestion
    setFighter(prevState=> prevState.slice(2);
```
state에 직접 접근하는 것이 권장되는 방식은 아니어서 `fighter`라는 state에 직접 접근하기 보단 prevState를 통해 접근하는 편이 더 좋을 것 같습니다! (state값을 수정하기 위해선 오로지 setState값을 쓰는 것만이 권장되고 있어요!)